_includes/syllabus.html

```diff
@@ -33,12 +38,14 @@ <h2 id="schedule">Schedule</h2>
           <td class="col-md-7"></td>
         </tr>
       {% endif %}
-      {% assign current = site.start_time %} {% comment %}Re-set start time of this episode to general daily start time {% endcomment %}
+      {% if partnotday %}{% else %}
```

Comment:
Would this break the multi day?

Reply:
Time must be reset here because this section is for a new day (see [line 29 above](https://github.com/carpentries/styles/pull/170/files#diff-6be1ef7cb9e3d5c3bd34049c20599594R29)).